final_task/rss_reader/rss_parser.py

```diff
@@ -0,0 +1,325 @@
+import feedparser
+import json
+import logging
+import psycopg2
+from yattag import Doc
+from database_functions import checkExistance
+from database_functions import createTable
+from database_functions import putIntoDB
+from database_functions import readingPasswordFromFile
+
+class RssParser():
+    """ Parses news from website"""
+
+    url = ""
+    limit = 0
+    date = None
+    jsonFormat = False
+    path = None
+    to_html = False
```

Comment:
You do not really need these class attributes if you are going to change them in the `__init__` anyway.

Reply:
Fixed